src/commands/src/shared/next-gen-stage.ts

```diff
@@ -0,0 +1,46 @@
+import { executeCliCommand, getCommitHash, getPatchId } from '../helpers';
+
+interface StageParams {
+  repoDir: string;
+  mutPrefix: string;
+  projectName: string;
+  bucket: string;
+  url: string;
+  patchId?: string;
+  commitBranch: string;
+}
+
+export async function nextGenStage({
+  repoDir,
+  mutPrefix,
+  projectName,
+  bucket,
+  url,
+  patchId,
+  commitBranch,
+}: StageParams) {
+  let hostedAtUrl = `${url}/${mutPrefix}/docsworker/${commitBranch}/`;
```

Comment:
Should `docsworker` be `docsworker-xlarge` to maintain existing staging url path? Or are we intentionally changing it here?

Reply:
Ah, I changed this for the local testing, but you're right. I'll need to rename it to `docsworker-xlarge` for when we use regular builds with these functions

Reply:
nit: would we want to assign `docsworker` to a constant since it is used in two places?